instrumentation/rails/test/test_helpers/middlewares/redirect_middleware.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+class RedirectMiddleware
+  def initialize(app, _options = {})
+    @app = app
+  end
+
+  def call(env)
+    return [307, {}, 'Temporary Redirect'] if env['PATH_INFO'] == '/redirection'
```

Comment:
For the 🥜 gallery, this represents something we've seen in a production app at Shopify 😄. Really anything would be fine for this test - it just represents middleware that returns early, without calling the `@app`.